timefhuman/categorize.py

```diff
@@ -37,6 +38,24 @@ def categorize(tokens, now):
     return tokens
 
 
+# TODO: add conversions for thirty, fourty-five
```

Comment:
These conversions could actually be done by https://pypi.org/project/word2number/